thaliproject/jxcore-for-thali
# add custom wrap

to test custom wrap;
./configure
make
./jx -e "var q=process.binding('custom_wrap').testMethod();
console.log(q)"

the output should be HelloWorld.
